vexim/admingroupchange.php

```diff
@@ -106,7 +106,7 @@ class="textfield">
                   </a>
                 </td>
                 <td><?php echo $row['realname']; ?></td>
-                <td><?php echo $row['username']; ?></td>
+                <td><?php echo $row['localpart'].'@'.$_SESSION['domain']; ?></td>
```

Comment:
This is what I thought at a first time, but later I saw that it's only allowed to add users in the same domain as the group and that we already have that domain name in `$_SESSION["domain"]` as it's already used in the same file in https://github.com/ultreia-es/vexim2/blob/dafd10b/vexim/admingroupchange.php#L45 so there is no need to add more tables to SQL query.